easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -40,10 +42,37 @@
 from easybuild.easyblocks.python import EXTS_FILTER_PYTHON_PACKAGES
 from easybuild.framework.easyconfig import CUSTOM
 from easybuild.framework.extensioneasyblock import ExtensionEasyBlock
-from easybuild.tools.filetools import mkdir, rmtree2, run_cmd
+from easybuild.tools.filetools import mkdir, rmtree2, run_cmd, write_file
 from easybuild.tools.modules import get_software_version
 
 
+# test setup.py script for PythonPackage.python_safe_install
+TEST_SETUP_PY = """#!/usr/bin/env python
+try:
```

Comment:
not necessarily, e.g. when installing EasyBuild with EasyBuild, we're typically using the system Python

and even if we think we're using a EB-installed Python with an EB-installed setuptools, another setuptools may 'win' (e.g. one installed with `--user`)

defining `$PYTHONNOUSERSITE` would already help there, though...